src/main/java/jeresources/profiling/Profiler.java

```diff
@@ -34,11 +35,20 @@ private Profiler(ICommandSender sender, int chunkCount, boolean allWorlds) {
     public void run() {
         LogHelper.warn("There will be messages about world gen lag during the profiling, you can ignore these as that is what you get when profiling.");
         if (!allWorlds) {
-            WorldServer world = (WorldServer) this.sender.getEntityWorld();
-            profileWorld(world);
-        } else {
-            for (WorldServer world : DimensionManager.getWorlds()) {
-                profileWorld(world);
+            
+            //What does this do if the profile command is run from the console?            
```

Comment:
You might want to figure this one out before I pull it in. Just to be safe.

Reply:
Seems to not be registered as a valid command for the console, I got a message saying it was an unknown.

Reply:
Oh yeah, JER is client side only. Let me derp out of here.